module/apps/actor-importer-dialog.js

```diff
@@ -99,6 +99,19 @@ export class CoC7ActorImporterDialog extends Dialog {
   }
 
   submit (button) {
+    if(button.cssClass=="getExampleNow"){
+      var content = CoC7ActorImporterRegExp.getExampleText($("#coc-entity-lang :selected").val());
+      navigator.clipboard.writeText(content)
```

Comment:
Please include CoC7Utilities and use `CoC7Utilities.copyToClipboard(content)` instead of `navigator.clipboard.writeText(content)` as this does not work for http requests over the internet.
This also handles the showing the errors messages.

Reply:
fixed